extensions-core/azure-extensions/src/main/java/org/apache/druid/storage/azure/AzureByteSourceFactory.java

```diff
@@ -26,5 +26,9 @@
  */
 public interface AzureByteSourceFactory
 {
-  AzureByteSource create(@Assisted("containerName") String containerName, @Assisted("blobPath") String blobPath);
+  AzureByteSource create(
+      @Assisted("containerName") String containerName,
```

Comment:
Do you need these `@Assisted` annotations here? The other factory interfaces dont seem to have them?

Reply:
i'm guessing thats why the assisted annotations are ther? i was just following the precedent in the code here